src/scs/scs.md

```diff
@@ -1,3 +1,7 @@
 # SCS
```

Comment:
Let's make the full name here.

Reply:
Agree. Better always open acronyms on topic level.